config/routes.rb

```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
+  # get 'users#index'
+  get '/users/:id', to: 'users#show'
+  get '/users/:user_id/posts', to: 'posts#index'
+  get '/users/:user_id/posts/:id', to: 'posts#show'
+  get '/users', to: 'users#index'
```

Comment:
- [ ] [Optional] instead of writing each routes individually, you can make it cleaner by writing in this format 
```
 resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```